src/Utilities/Extensions/IOperationExtensions.cs

```diff
@@ -153,5 +153,12 @@ public static ImmutableArray<IOperation> GetOperations(this ImmutableArray<IOper
                 return blockOperations;
             }
         }
+
+        public static bool IsAttribute(this IOperation operation)
+        {
```

Comment:
isn't this a bit too naïve way to check whether it is Attribute?

Reply:
Yes, but do we have other option without making it syntax specific check? Any other suggestions are welcome :)